src/cliff.jl

```diff
@@ -0,0 +1,30 @@
+#   The "cliff problem" in 2 variables
+#
+#   Source:  problem 206 (p. 46) in
+#   A.R. Buckley,
+#   "Test functions for unconstrained minimization",
+#   TR 1989CS-3, Mathematics, statistics and computing centre,
+#   Dalhousie University, Halifax (CDN), 1989.
```

Comment:
We haven't found it either but it's referenced everywhere. Instead I found https://doi.org/10.1145/138351.138378, which might be the published version (not sure).